.pylintrc

```diff
@@ -0,0 +1,597 @@
+[MASTER]
```

Comment:
if theres any lines that have nothing on the right side of the equals sign, remove it and the associated comments, lets make this file small